addon-test-support/index.ts

```diff
@@ -0,0 +1,8 @@
+export { default as a11yAudit } from './audit';
+export { default as a11yAuditIf } from './audit-if';
+export { invokeAll } from './invocation-strategies';
+export { setRunOptions, getRunOptions } from './run-options';
+export {
+  setupGlobalA11yHooks,
+  teardownGlobalA11yHooks,
+} from './setup-global-a11y-hooks';
```

Comment:
Just to better my knowledge, what is the reason for having to re-export these modules? Thanks!

Reply:
We re-export in the index of the addon (or in this case, the addon's `addon-test-support` dir), so that you can reference those imports using a single, public path. This means you won't have to 'dig in' to the module to find specific things to import, but rather the index file indicates what's publicly available. It also makes import paths a bit cleaner. Does this make sense?

Reply:
Yes, totally. Thanks for explaining!